src/Components/NavBar/NavItem.tsx

```diff
@@ -36,6 +40,8 @@ export const NavItem: React.FC<NavItemProps> = ({
   const getAnimation = h => ({
     opacity: h ? 0 : 1,
     transform: `translate3d(0, ${h ? -90 : -65}px, 0)`,
+    position: isFullScreenDropDown ? ("absolute" as Position) : "static",
```

Comment:
Nice, so this is how it was done 👍

Reply:
Is `"absolute" as Position` needed?

Reply:
I get the following type errors without it: `Types of property 'position' are incompatible. Type 'string' is not assignable to type 'PositionProperty'.`